src/docs/styles/docs-global.scss

```diff
@@ -59,7 +59,8 @@ main {
 }
 
 // Code fonts
-p code {
+p code,
```

Comment:
Does it make sense to have same style for `code` everywhere? It's not a common tag.

Reply:
Actually in the docs it's quite a common tag, it appears in most components, utility classes and styling related pages. Or I am missing something?